projects/pastanaga-angular/src/lib/controls/textfield/input/deprected-input.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="pa-field"
```

Comment:
typo in the file name: `deprected-input.component.html ` instead of `deprecated-input.component.html `